packer/scripts/prepare-offline-installation.sh

```diff
@@ -30,6 +30,9 @@ socat
 patch
 python-backports.ssl-match-hostname
 python-docker
+python-setuptools
+python-pip
+python-wheel
```

Comment:
These shouldn't be required, as they are present in debian.cfg?

Reply:
Good point, I will update this now.

Reply:
I've now removed these unneeded lines.